shared/config.lua

```diff
@@ -1,3 +1,4 @@
+lib.locale()
```

Comment:
This is already defined in the manifest

https://github.com/Project-Sloth/ps-adminmenu/blob/main/fxmanifest.lua#L34

Reply:
Sorry, maybe it was my mistake because I didn't use the latest version of ox_lib